Submitted By:            Matt Burgess (matthew at linuxfromscratch dot org)
Date:                    2009-01-25
Initial Package Version: 2.6.28.1
Upstream Status:         From upstream
Origin:                  From upstream Git commit
Description:             Fixes a build failure in Kbd-1.15

diff -Naur linux-2.6.28.1.orig/include/linux/serial.h linux-2.6.28.1/include/linux/serial.h
--- linux-2.6.28.1.orig/include/linux/serial.h	2009-01-18 18:45:37.000000000 +0000
+++ linux-2.6.28.1/include/linux/serial.h	2009-01-25 19:15:17.000000000 +0000
@@ -10,8 +10,9 @@
 #ifndef _LINUX_SERIAL_H
 #define _LINUX_SERIAL_H
 
-#ifdef __KERNEL__
 #include <linux/types.h>
+
+#ifdef __KERNEL__
 #include <asm/page.h>
 
 /*
